src/utils/loadSBML.cpp

```diff
@@ -37,13 +37,13 @@ int main(int argc, char** argv)
 
   SBMLDocument* document = reader.readSBML(filename);
   if (document == nullptr) {
-    std::cout << "Faile to get SBML document from " << filename << std::endl;
+    std::cout << "Failed to get SBML document from " << filename << std::endl;
     return -1;
```

Comment:
For `return -1`, use `return EXIT_FAILURE`.
For `return 0`, use `return EXIT_SUCCESS`.